pages/translation.zh-hans.html

```diff
@@ -47,25 +47,25 @@
 <header>
   	<nav id="mainNavigation"></nav><script>document.getElementById('mainNavigation').innerHTML = mainNavigation</script>
 	</header>
-<h1>Translation instructions</h1>
+<h1>翻译说明</h1>
 
 <section id="sidebarExtras">
 </section>
 
 
 <div id="mainLayout">
 <section>
-    <p>For ideas on what to translate, take look at the list of <a href="https://www.w3.org/International/articlelist">Articles, best practices &amp; tutorials</a>, select your language at the top right, and look for links that are still in English after the page has refreshed.</p>
+    <p>想知道能翻译哪些内容，请查看<a href="https://www.w3.org/International/articlelist">文章、最佳实践和教程</a>的列表，在右上角选择您的语言，并在页面更新后寻找标题为英文的链接。</p>
 </section>
 
 
 
 
 
 <section>
-    <h2 id="specs"><a href="#specs">W3C Specifications</a></h2>
+    <h2 id="specs">W3C规范文档</h2>
```

Comment:
id needs to go on the section tag

Reply:
Looks like that applies to all section headings

Reply:
Fixed both the translation and the English version. Thanks.